content/applications/general/in_app_purchase.rst

```diff
@@ -1,59 +1,159 @@
-   :align: center
+   .. image:: in_app_purchase/sms-icon.png
+      :align: center
+      :alt: Hover the mouse to the right of a contact's phone number to reveal the SMS phone icon.
+
+   1. Go to :menuselection:`Contacts app --> Click on a contact.`
+   2. Hover the mouse just to right of the phone number field until a :guilabel:`SMS 📱 (phone)`
+      icon appears.
+   3. Click the :guilabel:`SMS 📱 (phone)` icon.
+
+   .. image:: in_app_purchase/send-sms.png
+      :align: center
+      :alt: The Send SMS Text Message service can be accessed from a contact's record.
+
+   4. In the :guilabel:`Send SMS Text Message` pop-up that appears, type a message and click
+      :guilabel:`Send SMS`. Odoo will send the message via SMS and log what was sent in the chatter.
+
+   The prepaid credits for the SMS service are automatically deducted from the existing pool. If
+   there are not enough credits to send the message, Odoo prompts the purchase of more.
+
+For more information on how to use various IAP services, review the documentation below:
+
+.. seealso::
+   - :doc:`Lead mining </applications/sales/crm/acquire_leads/lead_mining>`
+   - :doc:`Enrich your contacts base with Partner Autocomplete
+     </applications/sales/crm/optimize/partner_autocomplete>`
+   - :doc:`SMS essentials </applications/marketing/sms_marketing/essentials/sms_essentials>`
+
+.. _in_app_purchase/credits:
+
+IAP credits
+===========
+
+Every time an IAP service is used, the prepaid credits for that service are spent. Odoo prompts the
+purchase of more credits when there are not enough credits to continue using the service. Email
+alerts can also be set up for when :ref:`credits are low <in_app_purchase/low-credits>`.
+
+Credits are purchased in "packs" from the `Odoo IAP website
+<https://iap.odoo.com/iap/all-in-app-services>`_, and pricing is specific to each service.
+
+.. example::
+   The `SMS service <https://iap.odoo.com/iap/in-app-services/1>`_ has four packs available for
+   purchase, in denominations of:
+
+   - **10** credits
+   - **100** credits
+   - **500** credits
+   - **1,000** credits
+
+   The number of credits consumed depends on the length of the SMS and the country of destination.
+   For more information on SMS pricing, click :doc:`here
+   </applications/marketing/sms_marketing/pricing/pricing_and_faq>`.
+
+.. _iap/buying_credits:
+
+Buy credits
+-----------
+
+When a service is used and there are not enough credit to perform the task, the database prompts the
+purchase of more credits, but users can check the current balance of credits for each service and
+even manually purchase more credits through the service's :guilabel:`Account Information` page.
+
+**To manually buy credits:**
+
+   .. image:: in_app_purchase/view-services.png
+      :align: center
+      :alt: The Settings app showing the Odoo IAP heading and View My Services button.
+
+#. Go to :menuselection:`Settings app --> search IAP --> View My Services --> Click on the
+   account`.
```

Comment:
This is a 17.0 vs 16.0 difference, I'm going to hold off on making this change until I determine if we can swap this commit to 17 or if we need to stick with 16.

Reply:
I can't seem to make a suggestion since this was already a comment, but I noticed in 17.0, it does not open in a new tab- it remains in the current tab, FYI.

Reply:
I would also mention it displays the information in a list view- in 16 I remember it being in boxes, but now all the runbots "do not have any iap's" installed so I can't check.